assets/js/admin/discounts/index.js

```diff
@@ -16,6 +16,10 @@ jQueryReady( () => {
 	 * Show/hide conditions based on input value.
 	 */
 	products.change( function() {
-		$( '#edd-discount-product-conditions' ).show( products.val() );
+		if (products.val() !== null) {
+			$( '#edd-discount-product-conditions' ).show( products.val() );
+		} else if (products.val() === null ) {
```

Comment:
Could this just be `else` ?

Reply:
@LisaCee The check for null can be done inside of `.toggle()`, like:

```js
$( '#edd-discount-product-conditions' ).toggle( null !== products.val() );
```

If condition side of `.toggle()` is `true` then the selector will to show, if it is false it will hide.

Reply:
Much more elegant.  Thanks